Assets/ModelReplacementSDK/Plugins/OffsetBuilder.cs

```diff
@@ -198,16 +199,26 @@ private IEnumerator SaveAssetBundle()
             initialized = true;
         }
 
-        private void MapSkeletonBones(SkeletonBone sk)
+        private void ForceRestPose(Animator animator)
         {
-            var matchingTransforms = GetComponentsInChildren<Transform>().Where(x => x.name == sk.name);
-            if (matchingTransforms.Any())
+            IEnumerable<Transform> childTransforms = animator.transform.root.GetComponentsInChildren<Transform>()
+                .Skip(1); // Skip root transform
+            foreach (var sk in animator.avatar.humanDescription.skeleton.Skip(1)) // Skip root bone
```

Comment:
Unity will also add the root transform to the skeleton definition. It needs to be skipped for three reasons:
1. This transform can't be rotated because it holds the entire prefab, not just the skeleton.
1. This GameObject can be renamed by the user since it is the top-level object of the prefab, so it won't always match.
1. The `SkeletonBone` definition will likely have "(Clone)" appended to the name preventing it from being matched to the actual transform anyways. This is due to how Unity's avatar editor works.